Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM smartcosmos/service
+MAINTAINER SMART COSMOS Platform Core Team
+
+ADD target/smartcosmos-*.jar  /opt/smartcosmos/smartcosmos-events.jar
+
+CMD ["java", "-jar", "/opt/smartcosmos/smartcosmos-events.jar"]
```

Comment:
I changed the base image, this should now look like:
`CMD ["/opt/smartcosmos/smartcosmos-events.jar"]`